modules/c++/math/wscript

```diff
@@ -2,6 +2,7 @@ NAME            = 'math'
 MAINTAINER      = 'asylvest@users.sourceforge.net'
 VERSION         = '0.1'
 USELIB          = 'MATH'
+MODULE_DEPS     = 'sys'
```

Comment:
Change this to TEST_DEPS = 'sys'.  This will make only the test depend on sys but not the module itself.